nmpc_controller/test/test_adaptive_complexity.cpp

```diff
@@ -0,0 +1,134 @@
+#include <gtest/gtest.h>
```

Comment:
This file probably needs some more comments

Reply:
Done.